README.md

```diff
@@ -20,7 +20,11 @@ To install packages run
 
 To run locally in development mode run
 
-    $ APP_ENV=development npm start
+    $ npm start
+
+The default environment for APIs is development. If you would like to update the environmenrt, run
+
+    $ APP_ENV=[environment] npm start
```

Comment:
What I mean is that it doesn't actually change `APP_ENV` when you run the command.